src/tlo/methods/rti.py

```diff
@@ -42,6 +42,7 @@ def __init__(self, name=None, resourcefilepath=None):
         self.resourcefilepath = resourcefilepath
         self.ASSIGN_INJURIES_AND_DALY_CHANGES = None
         self.item_codes_for_consumables_required = dict()
+        self.cons_item_codes = None  # dict containing the item_codes for consumables needed in the HSI Events.
```

Comment:
no need for this change (I don't think, so roll back?)

Reply:
This was for the change I had suggested. I should have removed it. I will roll back